spec/models/ability_spec.rb

```diff
@@ -210,7 +208,7 @@
       it{ should_not be_able_to(:manage, my_conference.tickets.first) }
       it{ should_not be_able_to(:manage, conference_public.tickets.first) }
 
-      it{ should_not be_able_to(:manage, my_registration) }
+      it{ should be_able_to(:manage, my_registration) }
```

Comment:
OK test was failing, but when a test fails, we don't change the test in order to make it pass. We search to find out what's wrong and a test that was succeeding, now fails.
Have you read a few chapters of https://github.com/CanCanCommunity/cancancan/wiki ? I am obviously more than happy to spend time to take you through whatever you don't get, however I cannot do that unless you first read up and experiment a bit with how cancancan works. When you do, find me on IRC, we will set up some time to discuss in more detail.